ERCS/erc-7007.md

```diff
@@ -1,10 +1,10 @@
 ---
 eip: 7007
-title: Zero-Knowledge AI-Generated Content Token
-description: An ERC-721 extension interface for zkML based AIGC-NFTs.
-author: Cathie So (@socathie), Xiaohang Yu (@xhyumiracle), Huaizhe Xu (@HuaizheXu), Kartin <kartin@hyperoracle.io>
+title: Verifiable AI-Generated Content Token
+description: An ERC-721 extension interface for zkML/opML-based AIGC-NFTs.
```

Comment:
Your description has a lot of abbreviations and probably isn't accessible for the average cryptocurrency developer. Expanding them here might be nice, but you're also limited on characters so I'll leave this up to your judgment.